showmore_dyn/lang/de/strings.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+if(! function_exists("string_plural_select_de")) {
```

Comment:
Did you generate this file yourself? The expected translation workflow for addons is the following:
- You run `bin/run_xgettext.sh --addon showmore_dyn` from Friendica root
- You commit the created `messages.po`
- Translation is done at Transifex
- Translated files are imported back into the addon